pkg/cli.go

```diff
@@ -52,63 +56,70 @@ func NewApp() *cli.App {
 					},
 				},
 			},
-		},
-		Flags: []cli.Flag{
-			&cli.StringFlag{
-				Name:    "template",
-				Aliases: []string{"t"},
-				Usage:   "template to use",
-			},
-		},
-		Action: func(c *cli.Context) error {
-			// Check if default template directory exists
-			_, err := os.Stat(absPath(DEFAULT_TEMPLATES_DIR))
-			if os.IsNotExist(err) {
-				fmt.Printf("Please run `jumpstart template sync` to seed templates\n\n")
-				os.Exit(2)
-			}
-
-			tid := c.String("template")
-			if tid == "" {
-				fmt.Printf("-t | --template is required\n\n")
-				cli.ShowAppHelp(c)
-				os.Exit(2)
-			}
+			{
+				Name:      "create",
+				Usage:     "Create a new project with the specified supporeted template",
+				UsageText: "jumpstart create -t={supported template name} [project-name]",
```

Comment:
We should remove this line and use what the library generates

Reply:
done